README.md

```diff
@@ -100,6 +100,7 @@ docker run \
         -v openhab_addons:/openhab/addons \
         -v openhab_conf:/openhab/conf \
         -v openhab_userdata:/openhab/userdata \
+        -e "EXTRA_JAVA_OPTS=-Duser.timezone=Europe/YourTZCity" \

```

Comment:
Should we make it a concrete example, so the example can still be copy-pasted

Reply:
I was considering the same. I had Helsinki there for my setup. I can change it with updated PR if you are OK with that city, or in case you want  some other city, tell me.

Reply:
Maybe Berlin because of the German roots openHAB has?